src/tj2_bringup/config/larry/tags/crescendo-2024-br-f114_tags_PK.yaml

```diff
@@ -0,0 +1,100 @@
+# # Definitions of tags to detect
```

Comment:
I think this the file you want to keep. Please rename to `crescendo-2024-br-f114_tags.yaml`